test/core/sdk.spec.js

```diff
@@ -407,7 +409,8 @@ describe('SDK', () => {
               'https://api.uphold.com/v0/foo',
               'get',
               undefined,
-              { authorization: 'foo' }
+              { authorization: 'foo' },
+              { headers: { authorization: 'foo' } }
```

Comment:
According to the test name, this shouldn't be added, right?

Reply:
The test name was wrong and confusing. Updated.